### DISCUSSION TITLE
Welcome to bpftrace Discussions!

### DISCUSSION THREAD

**Discussion by danobi** [bpftrace/bpftrace#1690] [General]
Please post bpftrace questions / discussions here.

**Comment by faglanatus** [comment]
![image](https://user-images.githubusercontent.com/33163300/142271950-12575264-86eb-485d-b7fc-52d9f374a270.png)
please help me i m try do to tcpconnect.bt

**Reply by fbs** [reply]
please open a new discussion, don't recycle old ones.

**Comment by faglanatus** [comment]
ok sory for mistake

Le mer. 17 nov. 2021 à 22:39, bas smit ***@***.***> a écrit :

> please open a new discussion, don't recycle old ones.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/iovisor/bpftrace/discussions/1690#discussioncomment-1661039>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AH5AQJFS5BHLI627HV5DYCTUMQOI3ANCNFSM4XNDD3AA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 

*Cordialement*

*SOFFTWAR **ENGINEER*
*NETWORK ENGINEER*
*CERTIFIED Cisco **CCNP **| Fortinet NSE7*
*Natus Fagla +229 97170520*
